
No idea when this was introduced and for what - but it prevents the definition
of vital basic types needed thruout many kernel headers which breaks some
applications depending of the order the headers are included (e.g. dosemu).

  - Rene Rebe <reneÂ@exactcode.de>

--- libc/sysdeps/unix/sysv/linux/sys/kd.h.orig	2000-01-01 00:00:00.000000000 +0100
+++ libc/sysdeps/unix/sysv/linux/sys/kd.h	2004-12-12 11:10:46.905723872 +0100
@@ -20,9 +20,11 @@
 #define _SYS_KD_H	1
 
 /* Make sure the <linux/types.h> header is not loaded.  */
+/* but not on our systems -ReneR
 #ifndef _LINUX_TYPES_H
 # define _LINUX_TYPES_H	1
 #endif
+*/
 
 #include <linux/kd.h>
 
